portfolio/src/main/java/com/google/sps/servlets/models/Comment.java

```diff
@@ -34,7 +34,7 @@ public Comment(
     this.status = status;
   }
 
-  public static Comment convertEntityToComment(Entity entity) {
+  public static Comment fromEntity(Entity entity) {
     long id = entity.getKey().getId();
     long timestamp = (long) entity.getProperty("timestamp");
     String userId = (String) entity.getProperty("userId");
```

Comment:
comment about likes type not fixed

Reply:
I added a comment explaining why we have to convert it from long to int on line 57-58, this is an expected behavior and I don't think we can do anything differently, other than change the "likes" field in Comment object to be also long type